examples/author.adoc

```diff
@@ -0,0 +1,28 @@
+= Author macro
```

Comment:
Hi @getJv 

I missed this as I'm not watching here, would you be able to add a test for it, there's some examples here:

https://github.com/jenkins-infra/asciidoctor-jenkins-extensions/tree/master/spec

Thanks!